packages/block-editor/src/components/rich-text/index.js

```diff
@@ -526,7 +530,6 @@ function RichTextWrapper(
 
 	const content = (
 		<RichText
-			{ ...props }
```

Comment:
This seems to by what is breaking Alignment on Mobile.

By removing ` { ...props } ` we are missing some important Mobile props:

These are some of those printed con console. Not sure if there might be more:
Most of this is text styling information that we can not pass via CSS.

```js
deleteEnter: true
fontSize: 12
onBlur: undefined
placeholderTextColor: "#ccc"
rootTagsToEliminate: ["p"]
textAlign: "center".  // ---> Alignment info that was missing
underlineColorAndroid: "transparent"
```

Reply:
@etoledom What component should be receiving these props? The internal RichText component or the wrapper element itself?

Reply:
Cc @SergioEstevao

Reply:
Ok, figured it out, I added the props.

Reply:
> Cc SergioEstevao

Thanks for reeling people in Ella! Just a quick update that Sérgio is on his 3-months sabbatical and it's best to instead ping @etoledom (iOS), or @cameronvoell (Android) or @guarani (iOS) on RichText matters. Thanks!